src/main/java/mate/controller/DisplayAllCarsController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/displayAllCars")
+public class DisplayAllCarsController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllCarsController extends HttpServlet {
```
this also maybe change to GetAllCarsController, as in checklist
-GetAllDriversController
-GetAllManufacturersController